### ISSUE TITLE
Cmd-key + another key

### ISSUE THREAD

**Issue by RblSb** [Kode/Kha#667]
The combination of the keys Cmd (down) + Z (down-up) + Z (down-up) + Cmd (up) produces the following event log from the kha.input.Keyboard (html5):
> sendDownEvent,224
> sendDownEvent,90
> sendUpEvent,224

(224 - cmd, 90 - z)
Expected behavior:

> sendDownEvent,224
> sendDownEvent,90
> sendUpEvent,90
> sendDownEvent,90
> sendUpEvent,90
> sendUpEvent,224

This is not a browser bug, the keys are handled correctly in pure js. Something's wrong with ControllerBuilder?

And it seems that krom/osx does not respond to modifier keys. It makes sense to create a separate issue?

**Comment by sh-dave** [comment]
ControllerBuilder is for syncing network input, are you trying to do that? Or is it wrong without network as well?

**Comment by RblSb** [comment]
@sh-dave you're right, I looked wrong.
I rechecked the behavior and it turned out that when holding the cmd, in fact, there are not currently created onUp events for other keys, only onDown / onPress. My Cmd + Z + Z does not work because Kha blocks the repeated call onDown without onUp.
https://github.com/Kode/Kha/blob/master/Backends/HTML5/kha/SystemImpl.hx#L860
If Robert wants to handle this, I can do PR with an additional check on keyCode 224 (checked for Chrome / FF), but it's probably a rare exception and it's not needed in Kha, it's enough to implement it for myself.
